RubberduckTests/Refactoring/RenameTests.cs

```diff
@@ -1766,6 +1766,123 @@ MsgBox CStr(CranApple)
             tdo.MsgBox.Verify(m => m.NotifyWarn(It.IsAny<string>(), It.IsAny<string>()), Times.Never);
         }
 
+        #endregion
+        #region Rename UDT Tests
+
+        [Test]
+        [Category("Refactorings")]
+        [Category("Rename")]
+        public void RenameRefactoring_RenamePublicUDT()
+        {
+            var tdo = new RenameTestsDataObject(selection: "UserType", newName: "NewUserType");
+            var inputOutput = new RenameTestModuleDefinition("Module1", ComponentType.StandardModule)
+            {
+                Input =
+                    @"Option Explicit
+
+Public Type UserType|
```

Comment:
Not related to the PR but I really wished we had used that faux cursor idea everywhere else instead of hard-coding the selections in the other unit tests. I will make a note to adapt that for other unit tests I'll be working on.

Reply:
Ha! I was thinking to do *exactly* that for the AC tests!